odufrn_downloader/mixins/filters/LevenshteinMixin.py

```diff
@@ -45,17 +46,24 @@ def search_related(self, keyword: str, input_list: list) -> list:
             palavra-chave com a qual será feita a busca.
         input_list: list
             lista com os valores que irá verificar a similaridade com keyword.
+        split: bool
+            flag que indica se a palavra-chave deve ser dividida
 
         Retorno
         -------
         lista de valores com nome similares à palavra de interesse.
         """
         filter_list = []
         for item in input_list:
-            for word in item.split('-'):
-                ratio = self.levenshtein(
-                    [k for k in keyword], [d for d in word]
-                )
+            str1 = [k for k in keyword]
+            if split:
```

Comment:
O código:
```python
ratio = self.levenshtein(str1, [d for d in word])
if ratio > 0.87:
    filter_list.append(item)
```
Aparece nas duas partes do `if` (dentro do `if` e dentro do `else`). Não seria melhor colocar uma condição dentro do `if` de forma que não precise do`else`?

Reply:
Não sei como corrigir isso, porque dentro do if usa o for, e no else não seria o caso